# HG changeset patch
# Parent 9d729f3ad85eb694c344aa6402f78638b9f5650a
libxl: Disable blktap

libxl does not compile cleanly against the XenServer out-of-tree
blktap.  But libxl was already designed to be able to build without
blktap.  So just disable libxl+blktap for now, until we can get
stuff sorted out.

diff -r 9d729f3ad85e tools/Rules.mk
--- a/tools/Rules.mk
+++ b/tools/Rules.mk
@@ -31,11 +31,7 @@ CFLAGS_libxenvchan = -I$(XEN_LIBVCHAN)
 LDLIBS_libxenvchan = $(SHLIB_libxenctrl) $(SHLIB_libxenstore) -L$(XEN_LIBVCHAN) -lxenvchan
 SHLIB_libxenvchan  = -Wl,-rpath-link=$(XEN_LIBVCHAN)
 
-ifeq ($(CONFIG_Linux),y)
-LIBXL_BLKTAP = y
-else
 LIBXL_BLKTAP = n
-endif
 
 ifeq ($(LIBXL_BLKTAP),y)
 CFLAGS_libblktapctl = -I$(XEN_BLKTAP2)/control -I$(XEN_BLKTAP2)/include $(CFLAGS_include)
